### PR TITLE
jsonnet/control-plane.libsonnet: Remove etcd rules

### PR DESCRIPTION
**Pull request by lilic**
As per request we are moving the rules to CEO -> https://github.com/openshift/cluster-etcd-operator/pull/613.

@openshift/openshift-team-monitoring please take a look. 
cc @hexfusion

### CODE REVIEW

**Comment by lilic** [comment]
> make: *** [assets/control-plane/etcd-prometheus-rule.yaml] Error 1
{"component":"entrypoint","error":"wrapped process failed: exit status 2","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-06-21T12:25:02Z"} 

🤷‍♀️ didn't get a chance to look into this yet, if someone has a hint?

**Comment by simonpasquier** [comment]
Hmm it seems to fail on `assets/control-plane/etcd-prometheus-rule.yaml` which are exactly the rules you're removing, right?

**Comment by lilic** [comment]
Is it maybe checking https://github.com/openshift/cluster-monitoring-operator/blob/7168290cd66048c65ff8cf8fb20a5fba849eec7d/pkg/manifests/manifests.go#L231, and need to remove this as well?

**Comment by simonpasquier** [comment]
@lilic you've spotted it :)

**Comment by simonpasquier** [comment]
https://github.com/openshift/cluster-monitoring-operator/blob/8ec92e1c442dcca2a8d73e10303fd37543745835/Makefile#L17

**Comment by lilic** [comment]
Ah nice, that explains it!

**Comment by lilic** [comment]
I guess we need to delete the `ControlPlaneEtcdPrometheusRule` first if it exists.

**Comment by lilic** [comment]
@simonpasquier please take another look, thanks!

File: pkg/manifests/manifests_test.go [hunk 1]
```diff
@@ -1428,46 +1428,6 @@ func TestPrometheusK8sControlPlaneRulesFiltered(t *testing.T) {
 	}
 }
 
-func TestPrometheusEtcdRulesFiltered(t *testing.T) {
```
Comment (lilic) [inline pkg/manifests/manifests_test.go:1431]: No clue what this is, I think its legacy?

Comment (simonpasquier) [reply pkg/manifests/manifests_test.go:1431]: Haha, not completely until the etcd service monitor moves to CEO too :)
But yes it's fine to remove the tests.

https://github.com/openshift/cluster-monitoring-operator/blob/8ec92e1c442dcca2a8d73e10303fd37543745835/pkg/operator/operator.go#L641-L647

**Comment by simonpasquier** [comment]
/lgtm
/hold
Holding to let @lilic merge this whenever you want/need.

**Comment by lilic** [comment]
> Watchdog alert had missing intervals during the run, which may be a sign of a Prometheus outage in violation of the prometheus query SLO of 100% uptime during upgrade

/retest

**Comment by lilic** [comment]
/retest

unrelated failures

**Comment by lilic** [comment]
> alert KubePodCrashLooping fired for 2209 seconds with labels: {container="thanos-query", endpoint="https-main", job="kube-state-metrics", namespace="openshift-monitoring", pod="thanos-querier-d4d687cc-g4fzq", service="kube-state-metrics", severity="warning"}

I don't think its related to my change, but this keeps firing. 🤔

**Pull request merged**